<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileInfoDialog</class>
 <widget class="QDialog" name="FileInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File informations</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="fileNameLabel">
       <property name="text">
        <string>Filename:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="readableLabel">
       <property name="text">
        <string>Readable:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="createdLabel">
       <property name="text">
        <string>Created:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lastModifiedLabel">
       <property name="text">
        <string>Last modified:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ownerLabel">
       <property name="text">
        <string>Owner:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sizeLabel">
       <property name="text">
        <string>Size: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fpsLabel">
       <property name="text">
        <string>FPS:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="framesLabel">
       <property name="text">
        <string>Frames:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="frameSizeLabel">
       <property name="text">
        <string>Frame size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lengthLabel">
       <property name="text">
        <string>Length:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FileInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FileInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
